transform/snowflake-dbt/models/event_registry/event_registry.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+
+models:
+  - name: base_tracks
+    description: >
+      Daily event information. Doesn't contain rows for events on dates where 
+      no events of that type were submitted.
+
+    columns:
+      - name: id
+        description: A unique id for each event/date/source combination.
+        tests:
+          - unique
+          - not_null
+      - name: date_received_at
+        description: The date that events of the current type were received.
+        tests:
+          - not_null
+      - name: event
```

Comment:
@ifoukarakis , maybe just call it `event_table` and the one below `event` or `event_name`? `event_text` seems more appropriate for some event description field.

Reply:
These are actually columns of the base tables. I reused the same descriptions for the columns as in https://www.rudderstack.com/docs/destinations/warehouse-destinations/warehouse-schema/#table-tracks. Am I missing something here?

We can also discuss names for `event_registry` and `daily_event_stats` tables. I thought of using `events`, but such table exists in other schemas, so I thought of avoiding conflicting names.

Reply:
Yeah, not the best (initial) naming imo, but better be consistent in this case I guess.

Reply:
@catalintomai what about the following:
- Rename `event_registry` to `event_fact`.
- Rename column `event` to `event_table` or `table`.
- Rename `event_text` to `event_name` to `name`.

Alternatively, we can use the naming above but with removing the `event_` prefix. 

`daily_event_stats` should keep the same name, but follow the same conventions.

Reply:
Thanks @ifoukarakis , that makes more sense/easier to follow to me (prefer `event_*` to `*` ).